From 204406df09b45e1316f02f1f629ef79574530b3d Mon Sep 17 00:00:00 2001
From: Michael Schroeder <mls@suse.de>
Date: Tue, 4 Aug 2015 14:30:04 +0200
Subject: [PATCH] Specify PYTHONLIBS_VERSION_STRING on the FIND_PACKAGE call

Otherwise the configure will fail for newer cmake versions, as
they expect _MAJOR/_MINOR/_COUNT to be set as well.
---
 bindings/python/CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/bindings/python/CMakeLists.txt b/bindings/python/CMakeLists.txt
index 8b5c72c..22ddec3 100644
--- a/bindings/python/CMakeLists.txt
+++ b/bindings/python/CMakeLists.txt
@@ -1,8 +1,7 @@
 #SET (PythonLibs_FIND_VERSION 3)
 
 FIND_PACKAGE (PythonLibs)
-SET (PythonInterp_FIND_VERSION ${PYTHONLIBS_VERSION_STRING})
-FIND_PACKAGE (PythonInterp REQUIRED)
+FIND_PACKAGE (PythonInterp ${PYTHONLIBS_VERSION_STRING} REQUIRED)
 EXECUTE_PROCESS(COMMAND ${PYTHON_EXECUTABLE} -c "from sys import stdout; from distutils import sysconfig; stdout.write(sysconfig.get_python_lib(True))" OUTPUT_VARIABLE PYTHON_INSTALL_DIR)
 
 IF (NOT DEFINED PYTHON_VERSION_MAJOR)
-- 
2.5.0

